great_expectations/profile/user_configurable_profiler.py

```diff
@@ -513,7 +536,8 @@ def _add_column_cardinality_to_column_info(self, profile_dataset, column_name):
 
         return column_cardinality
 
-    def _get_column_cardinality(self, profile_dataset, column):
+    @staticmethod
+    def _get_column_cardinality(profile_dataset, column):
```

Comment:
Same note here and below

Reply:
@cdkini -- added (in next commit).  Thanks!